### PR TITLE
Add ai/consulting page

### LINKED ISSUE #5083: [ai/consulting] Build on consulting page
per [copy doc](https://docs.google.com/document/d/19Mx1zRJKqH2hz5kRF_oDsgyFUw67FgJvW4I_OGda-wk/edit#heading=h.wtb1j7wwfobn)

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add ai/consulting page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/ai/consulting](http://0.0.0.0:8001/ai/consulting)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Got to the page and make sure it complies with the [copy doc](https://docs.google.com/document/d/19Mx1zRJKqH2hz5kRF_oDsgyFUw67FgJvW4I_OGda-wk/edit#heading=h.qo896jpcbdg0)

## Issue / Card

Fixes #5083

### CODE REVIEW

**Comment by pmahnke** [comment]
@solazio the contextual footer and contact-us includes are appearing?

**Comment by solazio** [comment]
@pmahnke I will remove them if they are not supposed to be there.

**Comment by anasereijo** [comment]
Hi @solazio see below some suggestions. There are new changes on the copydoc too.

- [ ] I think the secondary navigation is missing. Can you confirm this @pmahnke? see example below:
Screenshot 2019-05-02 at 14 29 34

- [ ] the arrow  `›` is missing from "Read our consulting datasheet ›" link

- [ ] "OUR CONSULTING PARTNERS" is part of the section above so it should be on a light grey background too

- [ ] Logos: we only have only 2 logos (for now): remove kubeflow logo

- [ ] `h4` looks too close to the text below (this applies to all the other sections too). Can we have some more space between the `h4` and the `p` text? I would like the h4 to be more as part of the title.

AI Consulting and Delivery   Ubuntu

**AI design sprint** section:

- [ ] please changes as per comments from the screenshot
AI Consulting and Delivery   Ubuntu (1)

**Learn more about AI/ML and Kubeflow** section

- [ ] all links are external so use `.p-link--external`

- [ ] make it look like this:

Screenshot 2019-05-02 at 15 31 47

my code skills are really bad but this how I've done. I didn't pay attention to clean code or anything like that so keep in mind **this is only a visual guide** (and links are missing etc) @anthonydillon please feel free to tell @solazio to ignore this if this is completely impossible to do (template wise or vanilla)

<section class="p-strip--light is-deep is-bordered">
    <div class="row">
      <div class="col-12">
        <h3>Learn more about AI/ML and Kubeflow</h3>
      </div>
    </div>
    <div class="row p-divider">
      <div class="col-6 p-divider__block">
        <div class="p-heading-icon--muted">
          <div class="p-heading-icon__header">
            
            <h4 class="p-heading-icon__title">Webinars</h4>
          </div>
        </div>
    
      <p>BootStack is your Kubernetes private cloud with our experts responsible for design, deployment and availability.</p>
    
      <p><a class="p-link--external">AI, ML, &amp; Ubuntu: Everything you need to know.</a></p>
      <p><a class="p-link--external">How to build and deploy your first AI/ML model on Ubuntu</a></p>    
      </div>

      <div class="col-6 p-divider__block">
        <div class="p-heading-icon--muted">
          <div class="p-heading-icon__header">
            
            <h4 class="p-heading-icon__title">Articles</h4>
          </div>
        </div>
      <p><a class="p-link--external">Stop Waiting on the AI Sidelines: Here's How to Take the Plunge</a>
</p>
      <p><a class="p-link--external">Getting Serious About AI</a>
</p>   
    
      <p><a class="p-link--external">Kubernetes and AI: Marriage Made in IT Heaven</a> 
</p>     
    
          <p><a class="p-link--external">Automating the Future of the Workplace</a> 
</p> 
      </div>
      
    </div>
  </section>

**Comment by anasereijo** [comment]
Hi @solazio see below some suggestions. There are new changes on the copydoc too.

- [ ] I think the secondary navigation is missing. Can you confirm this @pmahnke? see example below:
Screenshot 2019-05-02 at 14 29 34

- [ ] the arrow  `›` is missing from "Read our consulting datasheet ›" link

- [ ] "OUR CONSULTING PARTNERS" is part of the section above so it should be on a light grey background too

- [ ] Logos: we only have only 2 logos (for now): remove kubeflow logo

- [ ] `h4` looks too close to the text below (this applies to all the other sections too). Can we have some more space between the `h4` and the `p` text? I would like the h4 to be more as part of the title.

AI Consulting and Delivery   Ubuntu

**AI design sprint** section:

- [ ] please changes as per comments from the screenshot
AI Consulting and Delivery   Ubuntu (1)

**Learn more about AI/ML and Kubeflow** section

- [ ] all links are external so use `.p-link--external`

- [ ] make it look like this:

Screenshot 2019-05-02 at 15 31 47

my code skills are really bad but this how I've done. I didn't pay attention to clean code or anything like that so keep in mind **this is only a visual guide** (and links are missing etc) @anthonydillon please feel free to tell @solazio to ignore this if this is completely impossible to do (template wise or vanilla)
```

<section class="p-strip--light is-deep is-bordered">
    <div class="row">
      <div class="col-12">
        <h3>Learn more about AI/ML and Kubeflow</h3>
      </div>
    </div>
    <div class="row p-divider">
      <div class="col-6 p-divider__block">
        <div class="p-heading-icon--muted">
          <div class="p-heading-icon__header">
            <img class="p-heading-icon__img p-heading-icon__img--small" src="https://assets.ubuntu.com/v1/26797d10-newsfeed-stripvideo-icon.svg" width="32" alt="">
            <h4 class="p-heading-icon__title">Webinars</h4>
          </div>
        </div>
    
      <p>BootStack is your Kubernetes private cloud with our experts responsible for design, deployment and availability.</p>
    
      <p><a class="p-link--external">AI, ML, &amp; Ubuntu: Everything you need to know.</a></p>
      <p><a class="p-link--external">How to build and deploy your first AI/ML model on Ubuntu</a></p>    
      </div>

      <div class="col-6 p-divider__block">
        <div class="p-heading-icon--muted">
          <div class="p-heading-icon__header">
            <img class="p-heading-icon__img p-heading-icon__img--small" src="https://assets.ubuntu.com/v1/cd7a3a3c-white-paper-icon.svg" width="32" alt="">
            <h4 class="p-heading-icon__title">Articles</h4>
          </div>
```
        </div>
      <p><a class="p-link--external">Stop Waiting on the AI Sidelines: Here's How to Take the Plunge</a>
</p>
      <p><a class="p-link--external">Getting Serious About AI</a>
</p>   
    
      <p><a class="p-link--external">Kubernetes and AI: Marriage Made in IT Heaven</a> 
</p>     
    
          <p><a class="p-link--external">Automating the Future of the Workplace</a> 
</p> 
      </div>
      
    </div>
  </section>

**Comment by anasereijo** [comment]
@solazio a few more things:

- [ ] hero section
AI Consulting and Delivery   Ubuntu (2)

- [ ] Train your infrastructure engineers
AI Consulting and Delivery   Ubuntu (3)

File: templates/ai/consulting.html [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+      <p>
+        Our data science consulting partners bring machine learning and deep learning expertise to the AI mix and Canonical delivers  optimised infrastructure for multi-cloud AI with Ubuntu, GPGPUs, Kubernetes and Kubeflow. Assess data readiness, empower developers and accelerate your processing on-premises or in the cloud.
+      </p>
+      <p>
+        <a class="p-button--positive" href="/ai/contact-us?product=ai-index"
+          >Speak to the experts</a
+        >
+        <a
+          class="p-link--external"
+          href="https://assets.ubuntu.com/v1/223eab1a-canonical-kubernetes-enterprise-2018-25-07.pdf"
```
Comment (pmahnke) [inline templates/ai/consulting.html]: use the {{ ASSET_SERVER_URL }} token

File: templates/ai/consulting.html [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+        Together, we help deliver a five-day/five-step AI design sprint with your analytics and infrastructure teams. At the end of the engagement you will have a pattern for productive developer workstations, a machine learning infrastructure (in the cloud or on-premises), and AI applications delivering daily insights, powered by your data.
+      </p>
+      <p class="u-sv3">
+        With our structured program, we can approach any situation and provide the infrastructure and capabilities that your business needs. Proven best practices mean that we can take the guesswork and experimentation out of your AI adoption, and fast-track you to value without breaking your budget.
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <p class="u-align--center">OUR CONSULTING PARTNERS</p>
```
Comment (pmahnke) [inline templates/ai/consulting.html]: add `p-muted-heading` and make it an h5

File: templates/ai/consulting.html [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+      your AI transformation journey. Please note: this week typically relies on
+      having the right infrastructure in place to support the AI technology
+      stack.
+    </p>
+    <p class="p-heading--three u-sv3">
+      The AI design process in more detail
+    </p>
+  </div>
+  <div class="row">
+    <div class="col-6 u-hide--small u-align-center">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: I think we do these `col-12`

File: templates/ai/consulting.html [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+      <img src="{{ ASSET_SERVER_URL }}5d8aa78b-1+Discovery.svg" alt="" width="335px"/>
+    </div>
+    <div class="col-6">
+      <p class="p-heading--four">1 - Discovery</p>
+      <p>
+        Following a consultative whiteboarding session to set the stage, partner data scientists and data engineers
```
Comment (pmahnke) [inline templates/ai/consulting.html]: you don't have to do hard breaks in lines, it gets messy as you edit... you don't have to remove here, but in the future, don't bother

File: templates/ai/consulting.html [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+    </p>
+    <p class="u-sv3">
+      The key outcomes that you can expect from this week are a baseline
+      architecture for iterative solution development – which includes a working
+      continuous integration pipeline – and a high level strategy that addresses
+      your AI transformation journey. Please note: this week typically relies on
+      having the right infrastructure in place to support the AI technology
+      stack.
+    </p>
+    <p class="p-heading--three u-sv3">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: just use h3

File: templates/ai/consulting.html [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+        business applications that can add the most value.
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <hr class="p-separator"/>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p class="p-heading--four">2 - Assesment</p>
```
Comment (pmahnke) [inline templates/ai/consulting.html]: use h4

File: templates/ai/consulting.html [hunk 7]
```diff
@@ -0,0 +1,357 @@
...
+  </div>
+  <div class="row">
+    <hr class="p-separator"/>
+  </div>
+  <div class="row">
+    <div class="col-6 u-hide--small u-align-center">
+      <img src="{{ ASSET_SERVER_URL }}3f8aaa5f-3+Design.svg" alt="" width="335px"/>
+    </div>
+    <div class="col-6">
+      <p class="p-heading--four">3 - Design</p>
```
Comment (pmahnke) [inline templates/ai/consulting.html]: h4

File: templates/ai/consulting.html [hunk 8]
```diff
@@ -0,0 +1,357 @@
...
+        neural network model.
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <hr class="p-separator"/>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <p class="p-heading--four">4 - Implementation</p>
```
Comment (pmahnke) [inline templates/ai/consulting.html]: h4

File: templates/ai/consulting.html [hunk 9]
```diff
@@ -0,0 +1,357 @@
...
+  <div class="row">
+    <div class="col-6 u-hide--small u-align-center">
+      <img
+        src="{{ ASSET_SERVER_URL }}2021203f-5+Operation+and+feedback.svg"
+        alt="" 
+        width="335px"
+      />
+    </div>
+    <div class="col-6">
+      <p class="p-heading--four">5 - Operation and Feedback</p>
```
Comment (pmahnke) [inline templates/ai/consulting.html]: h4

File: templates/ai/consulting.html [hunk 10]
```diff
@@ -0,0 +1,357 @@
...
+        place on the rack or on GPGPU-accelerated instances on the cloud.
+        Resulting models are then delivered to the point of decision, whether
+        that’s on cloud or IoT at the edge.
+      </p>
+      <p>
+        Working with Canonical and all the leading clouds, Kubeflow on Ubuntu
+        provides infrastructure and operations that span that full range, and
+        seamless portability of AI workloads.
+      </p>
+      <a class="p-button--neutral" href="https://www.ubuntu.com/ai/features">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: remove the https://www.ubuntu/com bit, it is on this site.

File: templates/ai/consulting.html [hunk 11]
```diff
@@ -0,0 +1,357 @@
...
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Train your infrastructure engineers</h2>
+    <p>
+      Get more detailed information in our
+      <a
+        class="p-link--external"
+        href="https://assets.ubuntu.com/v1/223eab1a-canonical-kubernetes-enterprise-2018-25-07.pdf"
```
Comment (pmahnke) [inline templates/ai/consulting.html]: use the {{ ASSET_SERVER_URL }} token

File: templates/ai/consulting.html [hunk 12]
```diff
@@ -0,0 +1,357 @@
...
+        >consulting datasheet</a
+      >
+    </p>
+  </div>
+
+  {% include "shared/pricing/_kubernetes-consulting-add-ons.html" %}
+
+  <div class="row">
+    <p>
+      <a href="https://www.ubuntu.com/ai/contact-us?product=ai-index"
```
Comment (pmahnke) [inline templates/ai/consulting.html]: remove the https://www.ubuntu.com part

File: templates/ai/consulting.html [hunk 13]
```diff
@@ -0,0 +1,357 @@
...
+      Learn more about AI/ML and Kubeflow
+    </h3>
+  </div>  
+  <div class="row">
+    <div class="col-12">
+      <div class="row p-divider">
+        <div class="col-6 p-divider__block">
+            <div class="p-heading-icon--muted">
+              <div class="p-heading-icon__header">
+                <img class="p-heading-icon__img p-heading-icon__img--small" src="https://assets.ubuntu.com/v1/26797d10-newsfeed-stripvideo-icon.svg" width="32" alt="">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: use the {{ ASSET_SERVER_URL }} token

File: templates/ai/consulting.html [hunk 14]
```diff
@@ -0,0 +1,357 @@
...
+          <p>
+            <a class="p-link--external" href="https://blog.ubuntu.com/2018/10/02/ai-ml-model-ubuntu"
+              >How to build and deploy your first AI/ML model on Ubuntu</a
+            >
+          </p>
+        </div>
+        <div class="col-6 p-divider__block">
+          <div class="p-heading-icon--muted">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img p-heading-icon__img--small" src="https://assets.ubuntu.com/v1/cd7a3a3c-white-paper-icon.svg" width="32" alt="">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: use the {{ ASSET_SERVER_URL }} token

File: templates/ai/consulting.html [hunk 15]
```diff
@@ -0,0 +1,357 @@
...
+          <p>
+            <a
+              class="p-link--external"
+              href="https://www.cmswire.com/digital-experience/stop-waiting-on-the-ai-sidelines-heres-how-to-take-the-plunge/"
+              >Here's How to Take the Plunge</a
+            >
+          </p>
+          <p>
+            <a href="http://www.financederivative.com/getting-serious-about-ai/"
+              >Getting Serious About AI</a
```
Comment (pmahnke) [inline templates/ai/consulting.html]: p-link--external

File: templates/ai/consulting.html [hunk 16]
```diff
@@ -0,0 +1,357 @@
...
+{% block meta_copydoc %}https://docs.google.com/document/d/19Mx1zRJKqH2hz5kRF_oDsgyFUw67FgJvW4I_OGda-wk/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h1>
+        AI consulting and delivery
+      </h1>
+      <p class="p-heading--four u-sv3">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: h4

File: templates/ai/consulting.html [hunk 17]
```diff
@@ -0,0 +1,357 @@
...
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Develop your data science capabilities</h2>
+      <p class="p-heading--four u-sv3">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: h4

File: templates/ai/consulting.html [hunk 18]
```diff
@@ -0,0 +1,357 @@
...
+      <p>
+        Together, we help deliver a five-day/five-step AI design sprint with your analytics and infrastructure teams. At the end of the engagement you will have a pattern for productive developer workstations, a machine learning infrastructure (in the cloud or on-premises), and AI applications delivering daily insights, powered by your data.
+      </p>
+      <p class="u-sv3">
+        With our structured program, we can approach any situation and provide the infrastructure and capabilities that your business needs. Proven best practices mean that we can take the guesswork and experimentation out of your AI adoption, and fast-track you to value without breaking your budget.
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: col-12 u-align--center

File: templates/ai/consulting.html [hunk 19]
```diff
@@ -0,0 +1,357 @@
...
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>AI design sprint</h2>
+    <p class="p-heading--four u-sv3">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: h4

File: templates/ai/consulting.html [hunk 20]
```diff
@@ -0,0 +1,357 @@
...
+      <p class="p-heading--four">2 - Assesment</p>
+      <p>
+        Domain experts will provide initial
+        feedback based on the Discovery phase. They will report on the data and
+        make solution recommendations. They will discuss building a strategy
+        roadmap for the project based on your specific business case. Any
+        changes to your infrastructure should be highlighted during this step.
+      </p>
+    </div>
+    <div class="col-6 u-hide--small u-align-center">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: u-align--center

File: templates/ai/consulting.html [hunk 21]
```diff
@@ -0,0 +1,357 @@
...
+      <img src="{{ ASSET_SERVER_URL }}0bb09a0a-2+Assessment.svg" alt="" width="335px"/>
+    </div>
+  </div>
+  <div class="row">
+    <hr class="p-separator"/>
+  </div>
+  <div class="row">
+    <div class="col-6 u-hide--small u-align-center">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: u-align--center

File: templates/ai/consulting.html [hunk 22]
```diff
@@ -0,0 +1,357 @@
...
+      <p>
+        Complete the design process and begin
+        training and testing your AI model until it reaches the desired accuracy
+        threshold. Build a pipeline that will put your model into a suitable
+        environment for testing and feedback from additional stakeholders.
+        Domain experts will offer guidance on assessing machine learning
+        predictions and putting discovered insights into action.
+      </p>
+    </div>
+    <div class="col-6 u-hide--small u-align-center">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: u-align--center

File: templates/ai/consulting.html [hunk 23]
```diff
@@ -0,0 +1,357 @@
...
+      <img src="{{ ASSET_SERVER_URL }}273fd9ee-4+Implementation.svg" alt="" width="335px"/>
+    </div>
+  </div>
+  <div class="row">
+    <hr class="p-separator"/>
+  </div>
+  <div class="row">
+    <div class="col-6 u-hide--small u-align-center">
```
Comment (pmahnke) [inline templates/ai/consulting.html]: u-align--center

**Comment by anasereijo** [comment]
- [ ] there are a few more copychanges. check copydoc

- [ ] logo strip
   - amend typo
   - it's looking a bit odd with only 2 logos. I suggest aligning to the left side (check [this example](https://www.ubuntu.com/openstack/what-is-openstack))
  - replace Mancep logo with[ this one](https://assets.ubuntu.com/v1/e9e4d429-Manceps.png)

AI Consulting and Delivery   Ubuntu (5)

**"I design process in more detail" section**

- [ ] first illustration is not centered

- [ ] fix typo
AI Consulting and Delivery   Ubuntu (6)

**Perfectly portable AI workloads section**

- [ ] make kubeflow logo 180px wide

**Learn more about AI/ML and Kubeflow section**

- [ ] make the background light-grey

**Comment by anasereijo** [comment]
- [ ] logo strip: 'partners' is still misspelt (I think the best way to prevent this is to always copy/paste the text)

- [ ] fix the title here

Screenshot 2019-05-03 at 12 18 26

File: templates/ai/consulting.html [hunk 24]
```diff
@@ -0,0 +1,349 @@
...
+      </h1>
+      <h4 class="u-sv3">
+        Canonical and leading data scientists team up to consult on the full
+        enterprise AI stack.
+      </h4>
+      <p>
+        Our data science consulting partners bring machine learning and deep learning expertise to the AI mix and Canonical delivers  optimised infrastructure for multi-cloud AI with Ubuntu, GPGPUs, Kubernetes and Kubeflow. Assess data readiness, empower developers and accelerate your processing on-premises or in the cloud.
+      </p>
+      <p>
+        <a class="p-button--positive" href="/ai/contact-us?product=ai-index"
```
Comment (pmahnke) [inline templates/ai/consulting.html]: you need the class for the modal contact us - js-invoke-modal

File: templates/ai/consulting.html [hunk 25]
```diff
@@ -0,0 +1,349 @@
...
+        >consulting datasheet</a
+      >
+    </p>
+  </div>
+
+  {% include "shared/pricing/_kubernetes-consulting-add-ons.html" %}
+
+  <div class="row">
+    <p>
+      <a href="/ai/contact-us?product=ai-index"
```
Comment (pmahnke) [inline templates/ai/consulting.html]: there is a contact us in the include, I think remove this and from the copy doc too.

**Pull request merged**